### DISCUSSION TITLE
'loading chat' forever

### DISCUSSION THREAD

**Discussion by leonamlvs** [Zain-ul-din/whatsapp-ai-bot#15] [Q&A] [answered]
I've followed all the steps to make it work, but I'm stuck on 'loading chat' in both `setup.exe` and `npx yarn dev`.
![bug](https://github.com/Zain-ul-din/whatsapp-ai-bot/assets/10386859/e8da66e8-bde6-409e-9d21-92f37580b5bd)

**Comment by Zain-ul-din** [comment]
hey thku for reaching out, please can you provide some information about what `OS`, `npm version` and `nodejs` version you are using?

**to get npm version** run `npm --version`
**to get node js version** run `node --version`


also, make sure you scan QR code properly

**Comment by Zain-ul-din** [comment, marked as answer]
i face same problem you can fix it by upgrading `whatsapp-web.js` package run

`npx yarn add whatsapp-web.js@latest`
or
`npm i whatsapp-web.js@latest --save`

## OR

reinstall latest code from github

**Reply by leonamlvs** [reply]
Thank you! It works :)

**Reply by Zain-ul-din** [reply]
good to hear! Enjoy :-). please give star this repo if you find this useful
